ludwig/typing.py

```diff
@@ -0,0 +1,28 @@
+"""Common typing for Ludwig dictionary parameters."""
+from typing import Any, Dict
+
+# Dictionary of parameters used to configure an input or output feature.
+# https://ludwig.ai/latest/configuration/features/supported_data_types/
+FeatureConfigDict = Dict[str, Any]
```

Comment:
Though there is not a standard way to provide a doc-string for a variable, Sphinx, VSCode, and PyCharm support this: (they will show the doc string when you hover over the variable name).  Maybe we should use this convention:
```
FeatureConfigDict = Dict[str, Any]
"""
Dictionary of parameters used to configure an input or output feature.
https://ludwig.ai/latest/configuration/features/supported_data_types/
"""
```

Reply:
Great idea! Done.